shared/elements/gaia_grid/js/items/group.js

```diff
@@ -36,6 +36,8 @@
     this.pendingCollapse = this.detail.collapsed = !!this.detail.collapsed;
   }
 
+  var windowWidth = window.innerWidth;
+
```

Comment:
Same concern here about conflating window-width and grid-width. If GridLayout doesn't have a gridWidth function, it might be nice to add one.